### PR TITLE
some changes to the test

### PR DESCRIPTION
**Pull request by ruslandoga**

### CODE REVIEW

File: test/ch/aggregation_test.exs [hunk 1]
```diff
@@ -19,10 +16,11 @@ defmodule Ch.AggregationTest do
     ORDER BY (ticker, time)
     """
 
-    Ch.query!(conn, create_table, %{})
+    Ch.query!(conn, create_table)
+    on_exit(fn -> Ch.Test.sql_exec("drop table ch_candle_fragments") end)
```
Comment (ruslandoga) [inline test/ch/aggregation_test.exs:20]: I change the order of creating and dropping the tables to ensure the test fails if some other contributor already has `candle_fragments` table defined. And I also prefix it with `ch_`. The proper fix is to create a database for `ch` tests like in `chto`, I'll do it tomorrow.

File: test/ch/aggregation_test.exs [hunk 2]
```diff
@@ -77,13 +76,13 @@ defmodule Ch.AggregationTest do
...
-    %{rows: rows} = Ch.query!(conn, query, %{})
+    %{rows: rows} = Ch.query!(conn, query)
 
     expected = [
       [
         "INTC",
-        ~U[2023-04-13 18:00:00Z],
-        ~U[2023-04-13 19:00:00Z],
+        ~U[2023-04-13 20:00:00Z],
+        ~U[2023-04-13 21:00:00Z],
```
Comment (ruslandoga) [inline test/ch/aggregation_test.exs:85]: `toStartOfHour(datetime, 'UTC')` I think (not sure) is equivalent to `toStartOfHour(toTimezone(datetime, 'UTC'))` and `toTimezone` just gives a different type name to the column, i.e. `DateTime('UTC')`. There isn't any actual timezone conversion happening in ClickHouse. So I think if you are inserting 

```sql
('INTC', '2023-04-13 20:33:00', 32, 32, 32, 32),
('INTC', '2023-04-13 20:34:00', 33, 33, 33, 33),
('INTC', '2023-04-13 20:35:00', 32, 32, 31, 26),
('INTC', '2023-04-13 20:36:00', 32, 27, 27, 27)
```
and then using `toStartOfHour(datetime, 'UTC')`, that's the same as assuming the inserted timestamps are UTC already. And then the aggregated rows should have 20:00 and 21:00 as time.

Comment (Zarathustra2) [reply test/ch/aggregation_test.exs:85]: The issue is that if we insert as string see https://clickhouse.com/docs/en/sql-reference/data-types/datetime

```
When inserting datetime as an integer, it is treated as Unix Timestamp (UTC). 1546300800 represents '2019-01-01 00:00:00' UTC. However, as timestamp column has Asia/Istanbul (UTC+3) timezone specified, when outputting as string the value will be shown as '2019-01-01 03:00:00'
When inserting string value as datetime, it is treated as being in column timezone. '2019-01-01 00:00:00' will be treated as being in Asia/Istanbul timezone and saved as 1546290000.
```

So now if someone runs the test locally it depends whether the column is set to UTC or not. It is better to insert these as unix timestamps

Comment (Zarathustra2) [reply test/ch/aggregation_test.exs:85]: I think my latest change fixed it

Comment (ruslandoga) [reply test/ch/aggregation_test.exs:85]: If possible, I'd prefer text datetimes, I think it's easier to understand. What if we set the column type to `DateTime('UTC')`?

Comment (Zarathustra2) [reply test/ch/aggregation_test.exs:85]: ahhh good point

Comment (ruslandoga) [reply test/ch/aggregation_test.exs:85]: like this: https://github.com/Zarathustra2/ch/pull/1/commits/eb3fea74784f70ce2d51cb8f39b4ab796b23b976, does it work?

Comment (Zarathustra2) [reply test/ch/aggregation_test.exs:85]: I will try and revert my changes

**Pull request merged**